package.json

```diff
@@ -13,6 +13,16 @@
 	"engines": {
 		"node": ">=14.18"
 	},
+	"main": "index.js",
+	"exports": {
+		".": {
+			"import": "./index.mjs",
+			"require": "./index.js"
+		},
+		"./all": "./configs/all.mjs",
+		"./recommended": "./configs/recommended.mjs",
+		"./package.json": "./package.json"
```

Comment:
package.json should not be exported.

Reply:
I heard some _old_ static analysis tools read package.json by `require()` not `fs.readFile(Sync)`. 
If such a tool runs on modern nodejs (so `require()` can only read what explicitly `exports`ed), it would not work. 

I personally do not have a strong opinion on this issue. How do you think?

Reply:
I perfer `./*` toexport everything.

Reply:
I think it's better to be explicit so we don't accidentally export something we don't want to be a public export.

Reply:
We can keep the package.json export until we move to ESM fully.

Reply:
If so, I don't understand why we need `./recommend` export, shouldn't ESLint load config via plugin entry?

Reply:
@fisker In the new system, eslint does not automatically load config from a plugin as well. 
Since `plugin:` protocol(e.g. `plugin:unicorn/recommended`) is no longer used in the new config system, it's not expected for a plugin to have `configs` property. That's why I created index.mjs differently from index.js.
Everything should be explicitly configured by nodejs's native module resolution.
Users should have a way to import sharable configs independently.

Reply:
Thanks for the explanation.